packages/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <div
+    v-if="isVisible"
+    :aria-label="title"
+    class="k-modal isOpen"
+    role="dialog"
+    aria-modal="true">
+    <div
+      class="k-modal-dialog"
+    >
+      <div class="k-modal-header">
+        <div
+          v-if="$scopedSlots.title || !hideTitle"
+          class="k-modal-header-description mb-5"
+          role="heading"
+          aria-level="2">
+          <div class="k-modal-title">
+            <span class="header-icon">
+              <slot name="header-icon"/>
+            </span>
+            <span class="header-content">
+              <slot name="header-content">{{ title }}</slot>
+            </span>
+          </div>
+          <div
+            class="k-modal-action ml-3">
+            <KButton
+              :appearance="cancelButtonAppearance"
+              @click="close"
+              @keyup.esc="close">
+              {{ cancelButtonText }}
+            </KButton>
+            <!-- @slot Use this slot to pass extra buttons other than Cancel  -->
+            <slot
+              v-if="hasActionButtons"
```

Comment:
@kaiarrowood: I am having trouble understanding this. Since we dont have a cancel button on both top/bottom, wouldn't it make sense to always have a cancel button on top and make other buttons optional?

Reply:
Thinking of the types of modal, you never have a modal with only a `Cancel` button, if there is only one action to take on the modal it would generally be to click `OK`. A user should always have a way to close the modal, and we only offer one button to close it, that button should be confirmation not cancel. Why would we open a modal where the only action that can be taken is to `Cancel`?